website/src/data/gallery.json

```diff
@@ -191,5 +191,12 @@
     "description": "X-Force IDE is a low-code, agent-as-a-service UI framework that lets you create AutoGen-based workforces from a drag-and-drop-like user interface.",
     "image": "x-force-ide-ui.png",
     "tags": ["ui"]
+  },
+  {
+    "title": "Multimodal Webagent created with AutoGen and OpenAI's Assistants API",
+    "link": "https://github.com/schauppi/MultimodalWebAgent",
+    "description": "A multimodal webbrowsing agent that autonomously browses the web.",
+    "image": "default.png",
```

Comment:
Can you provide an image? Gallery entries with the default don't look great

Reply:
This one would work https://camo.githubusercontent.com/7a317cc6897be1e55ca506140a0b6b66f7164a2ca6b910d6264a9b85b03f0837/68747470733a2f2f696d672e796f75747562652e636f6d2f76692f6a51324f733638325962732f302e6a7067